src/ReactiveUI.AndroidSupport/ControlFetcherMixin.cs

```diff
@@ -34,10 +34,9 @@ public static class ControlFetcherMixin
 
         static ControlFetcherMixin()
         {
-            // NB: This is some hacky shit, but on Xamarin.Android at the moment,
-            // this is always the entry assembly.
-            var assm = AppDomain.CurrentDomain.GetAssemblies()[1];
-            var resources = assm.GetModules().SelectMany(x => x.GetTypes()).First(x => x.Name == "Resource");
+            var assemblyName = Assembly.GetCallingAssembly().GetName().Name;
```

Comment:
There are a couple of issues, if any other method is called inside potentially could get the wrong assembly. Sorry noticed it more when I was back at the PC.